arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -919,7 +907,8 @@ Result IResearchDataStore::deleteDataStore() {
 }
 
 Result IResearchDataStore::initDataStore(
-    InitCallback const& initCallback, uint32_t version, bool sorted,
+    bool& pathExist, InitCallback const& initCallback, uint32_t version,
```

Comment:
NIT: pathExists looks like more correct name.
```suggestion
    bool& pathExists, InitCallback const& initCallback, uint32_t version,
```

Reply:
Yep, but clang format make code very ugly in some places

Reply:
Done